### DISCUSSION TITLE
Whisper multi language

### DISCUSSION THREAD

**Discussion by limnetik** [tin2tin/Pallaidium#103] [Ideas]
Hello, i am testing new things in Pallaidium and i was wondering if there is a way to specify the language of the speaker with whisper text to audio? Is there a switch or command to add. I am specifically interested in generating voices in french. I see bark works with French ok. I searched and for now the models are probably in training for french in whisper. But in prevision of an option, it would be nice to be able to specify the language like in Bark options. Thanks

**Comment by tin2tin** [comment]
Currently WhisperSpeech only supports 3 languages: en, fr, pl. I'll not enable more languages until there will be more supported languages: https://github.com/collabora/WhisperSpeech

You can download the pallaidium zip, unzip it, and change "en" to "fr" in ex notepad, save it and zip the folder, then install it, and test if it is working: 
https://github.com/tin2tin/Pallaidium/blob/4b5bf3d0f229c46df7852eae0d6ed840a96b9f89/__init__.py#L2915

**Reply by limnetik** [reply]
So cool, i'll try this soon! Thank you!
Your work on the plugin is truly inspiring! Keep on the good work!

**Reply by limnetik** [reply]
ok i tried and the language does switch but it does not sound like french and nor work for english. Maybe later, i'll retry or check if something pops out in a search on the subject. thanks
